### DISCUSSION TITLE
Possible to write a text file from the command line?

### DISCUSSION THREAD

**Discussion by 7k50** [openai/whisper#388] [Q&A] [answered]
Apologies if this has been answered previously, but I couldn't find a method.

I'm trying to figure out if there's any way to write a text file of the transcription output, when using the command line? E.g. based on:

`whisper "clip.mp3" --model medium.en --language English`

(Saving the Terminal output to a text file is not the solution I'm looking for, but rather to get Whisper to generate TXT/SRT/etc, or capturing the application output to a file).

**Comment by ResponsiveEcologiesLab** [comment]
I haven't found out the solution to this yet either. At present, I'm getting the output dumped into a terminal shell, and saving it as a text file. It is working for the first 20 minutes of the 90 minute file pretty well, but I'm assuming that there will be a full buffer and I'll lose it. Still, I'm close.  I was hoping it would be something like: whisper "clip.mp3" "textfile.txt" --model medium.en --language English

**Reply by ResponsiveEcologiesLab** [reply]
This link says it will create a text file in the same folder when it is done https://bytexd.com/how-to-use-whisper-a-free-speech-to-text-ai-tool-by-openai/ And I'm hoping that is the case.

**Comment by YaoYYoung** [comment, marked as answer]
If you type `whisper --help` in PowerShell, you can see below:

> usage: 
whisper ... [--output_dir OUTPUT_DIR] [--output_format {txt,vtt,srt,tsv,json,all}] ...
...


> options:
...
--output_dir OUTPUT_DIR, -o OUTPUT_DIR
    directory to save the outputs (default: .)
--output_format {txt,vtt,srt,tsv,json,all}, -f {txt,vtt,srt,tsv,json,all}
    format of the output file; if not specified, all available formats will be produced (default: all)
...

> 

So maybe you can do like below: 

`whisper.exe demo.mp4 --model medium --language 'Chinese' --output_format txt --task transcribe`

Hope useful.
